ctapipe/reco/tests/test_HillasReconstructor.py

```diff
@@ -72,87 +82,273 @@ def test_h_max_results():
     p2 = SkyCoord(alt=0 * u.deg, az=90 * u.deg, frame=horizon_frame)
     circle4 = HillasPlane(p1=p1, p2=p2, telescope_position=[-1, 0, 0] * u.m)
 
+    # Create a dummy subarray
+    # (not used here, but required to initialize the reconstructor)
+    subarray = SubarrayDescription(
```

Comment:
define this dummy subarray once as fixture. Or use one of the already existing fixtures.

Reply:
done